File: 1317. Convert Integer to the Sum of Two No-Zero Integers/solution.go
func getNoZeroIntegers(n int) []int {
	isNoZero := func(num int) bool {
		numStr := fmt.Sprint(num)
		if strings.Contains(numStr, "0") {
			return false
		}
		return true
	}

	for i := 1; i < n; i++ {
		right := n - i
		if isNoZero(i) && isNoZero(right) {
			return []int{i, right}
		}
	}

	return nil
}

File: 1317. Convert Integer to the Sum of Two No-Zero Integers/solution.js
/**
 * @param {number} n
 * @return {number[]}
 */
var getNoZeroIntegers = function (n) {
    const isNoZero = (n) => !`${n}`.includes("0");

    for (let i = 1; i < n; i++) {
        const diff = n - i;
        if (isNoZero(i) && isNoZero(diff)) {
            return [i, diff];
        }
    }
};
